modules/ROOT/pages/alternatives.adoc

```diff
@@ -1,37 +1,21 @@
 = Setting alternatives
 
-Due to an https://github.com/fedora-sysv/chkconfig/issues/9[ongoing issue] in how alternatives configurations are stored on the system, Fedora CoreOS systems can not use the usual `alternatives` commands to configure them.
+Now Fedora CoreOS systems can use the usual `alternatives` commands to configure the default command, as https://github.com/fedora-sysv/chkconfig/issues/9[issue] is fixed.
 
-Instead, until this issue is resolved, you can set the symlinks directly in `/etc/alternatives`. For example, to use the legacy-based variants of the `iptables` commands:
+For example, to use the legacy-based variants of the `iptables` commands:
 
-[source,yaml,subs="attributes"]
+[source,bash,subs="attributes"]
 ----
-variant: fcos
-version: {butane-latest-stable-spec}
-storage:
-  links:
-    - path: /etc/alternatives/iptables
-      target: /usr/sbin/iptables-legacy
-      overwrite: true
-      hard: false
-    - path: /etc/alternatives/iptables-restore
-      target: /usr/sbin/iptables-legacy-restore
-      overwrite: true
-      hard: false
-    - path: /etc/alternatives/iptables-save
-      target: /usr/sbin/iptables-legacy-save
-      overwrite: true
-      hard: false
-    - path: /etc/alternatives/ip6tables
-      target: /usr/sbin/ip6tables-legacy
-      overwrite: true
-      hard: false
-    - path: /etc/alternatives/ip6tables-restore
-      target: /usr/sbin/ip6tables-legacy-restore
-      overwrite: true
-      hard: false
-    - path: /etc/alternatives/ip6tables-save
-      target: /usr/sbin/ip6tables-legacy-save
-      overwrite: true
-      hard: false
+# The legacy priority is 10, update nft to lower priority, so legacy wins
+pfx=/usr/sbin/iptables
+pfx6=/usr/sbin/ip6tables
+sudo alternatives --install ${pfx} iptables ${pfx}-nft 5 \
```

Comment:
Hum, that looks weird. I'll give it a try.

Reply:
Thanks for the kind review!